src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java

```diff
@@ -111,7 +129,24 @@
  *   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_TRY&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   private void emptyLoop() {
+ *     for (int i = 0; i &lt; 10; i++) {
+ *       // ignored
+ *     }
  *
+ *     try {
+ *       // violation
+ *     } catch (Exception e) {
+ *       // ignored
+ *     }
+ *   }
+ * }
```

Comment:
```
/var/tmp/checkstyle $ cat Test.java 
public class Test {
  private void emptyLoop() {
    for (int i = 0; i < 10; i++) {
      // ignored
    }

    try {
      // violation
    } catch (Exception e) {
      // ignored
    }
  }
}
```
```
/var/tmp/checkstyle $ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
       <module name="EmptyBlock">
         <property name="option" value="text"/>
         <property name="tokens" value="LITERAL_TRY"/>
       </module>
    </module>
</module>
```
```
/var/tmp/checkstyle $ java -jar checkstyle-8.30-all.jar -c config.xml Test.java 
Starting audit...
Audit done.
```
In your example you say there is a violation but it is not true.

Reply:
fixed